### PR TITLE
Check for clock gettime

### PR DESCRIPTION
**Pull request by rubenk**

### CODE REVIEW

**Comment by pieterlexis** [comment]
This links everything against librt. As discussed, this needs improvement.

File: pdns/dnsdistdist/Makefile.am [hunk 1]
```diff
@@ -55,6 +55,9 @@ dnsdist_LDFLAGS = \
 	-pthread 
 
 dnsdist_LDADD = \
-	$(READLINE_LIBS) -lrt \
-	$(LUA_LIBS) $(YAHTTP_LIBS) ${LIBSODIUM_LIBS}
+	$(LUA_LIBS) \
+	$(READLINE_LIBS) \
+	$(RT_LIBS) \
+	$(YAHTTP_LIBS) \
+	${LIBSODIUM_LIBS}
```
Comment (pieterlexis) [inline pdns/dnsdistdist/Makefile.am]: Hrm, I just realized these braces are different than the rest (I think that is my doing) is that an issue?

**Comment by rubenk** [comment]
It's not an issue, as far as I know they're exactly the same, but I've fixed them up.

**Comment by pieterlexis** [comment]
:+1:

**Pull request merged**